### ISSUE TITLE
Compatability with globalize master

### ISSUE THREAD

**Issue by jhawthorn** [solidusio-contrib/solidus_globalize#9]
We should loosen the dependency on globalize

Ref: solidusio/solidus#1010

**Comment by modreoci** [comment]
Can somebody fix this issue, please?

**Comment by modreoci** [comment]
```
Resolving dependencies...
Bundler could not find compatible versions for gem "globalize":
  In Gemfile:
    globalize

    solidus_globalize was resolved to 3.1.0.beta, which depends on
      globalize (~> 5.0.1)

Could not find gem 'globalize (~> 5.0.1)', which is required by gem
'solidus_globalize', in any of the sources.
```

Part of Gemfile:

``` ruby
gem 'solidus'
gem 'solidus_auth_devise'
gem 'rails-i18n'
gem 'devise-i18n'
gem 'solidus_globalize', github: 'solidusio-contrib/solidus_globalize', branch: 'master'
gem 'globalize', github: 'globalize/globalize', branch: 'master'
gem 'solidus_i18n', github: 'solidusio-contrib/solidus_i18n', branch: 'master'
```

**Comment by rbngzlv** [comment]
Hi @jhawthorn, is possible that you have already made the change in this commit (https://github.com/solidusio-contrib/solidus_globalize/commit/1b5218ca4a3c24c6878325412cc267797185dde8) and you haven't remembered to close the issue?

**Comment by AndreiMotinga** [comment]
The problem that @modreoci was having should be addressed with #41

**Comment by mamhoff** [comment]
Closed via https://github.com/solidusio-contrib/solidus_globalize/pull/45/files
